server/auth/rest/auth_rest.go

```diff
@@ -239,6 +239,12 @@ func (a *authenticator) RestrictedTags() ([]string, error) {
 	return resp.StrSliceVal, nil
 }
 
+// GetResetParams returns authenticator parameters passed to password reset handler
+// (none for rest).
+func (authenticator) GetResetParams(uid types.Uid, authScheme string) (map[string]interface{}, error) {
```

Comment:
ditto `authScheme`

Reply:
It would be nice to pass request to the server here instead of returning nil, but it can be done later.

Reply:
Added a TODO.